homeassistant/components/tradfri/base_class.py

```diff
@@ -13,18 +13,18 @@
 from pytradfri.error import PytradfriError
 
 from homeassistant.core import callback
-from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity import DeviceInfo, Entity
 
 from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def handle_error(func):
+def handle_error(func: Callable) -> Callable:
     """Handle tradfri api call error."""
 
     @wraps(func)
-    async def wrapper(command):
+    async def wrapper(command: str) -> None:
```

Comment:
I think `command` will be a `Command` instance.

Reply:
Done a while ago, but forgot to resolve conversation.